### ISSUE TITLE
Feature request: Add IP geo location to api call. 

### ISSUE THREAD

**Issue by ChumpyTheBear** [qdm12/gluetun#601]
Would it be possible to integrate a service like https://ipgeolocation.io/ to the HTTP control server as an api call? This would be helpful to validate a VPN's config.  /v1/publicip/ip seems to only return the VPN's ip address without its geo location.

**Comment by nearcatch** [comment]
> Would it be possible to integrate a service like [ipgeolocation.io](https://ipgeolocation.io/) to the HTTP control server as an api call? This would be helpful to validate a VPN's config. /v1/publicip/ip seems to only return the VPN's ip address without its geo location.

IPGeolocation.io returns inaccurate or missing info for some locations I tested: it returns a Berlin address as Washington DC and doesn't return the province for Dublin. Getting that fixed is annoying because their contact form requires a phone number, which I think is unnecessary.

Also they limit the free api to 1000 requests/day. IPinfo.io seems more accurate from my tests and they give non-authorized users 1000 requests/day. If you sign up for the free api you get 50,000/month.

I think this would be a neat improvement to Gluetun's api, but in the meantime you can get this info from the commandline if that works for you:
```shell
curl -m 15 https://ipinfo.io/$(curl -m 15 GLUETUN:8000/v1/publicip/ip | jq -r '.public_ip')
```

gives this result (cloudflare ip used as example):
```json
{
  "ip": "1.1.1.1",
  "hostname": "one.one.one.one",
  "city": "Miami",
  "region": "Florida",
  "country": "US",
  "loc": "25.7867,-80.1800",
  "org": "AS13335 Cloudflare, Inc.",
  "postal": "33132",
  "timezone": "America/New_York",
  "readme": "https://ipinfo.io/missingauth"
}
```

**Comment by qdm12** [comment]
Thanks @nearcatch for the comment.

Actually also see my other comment in the discussion: https://github.com/qdm12/gluetun/discussions/600#discussioncomment-1280701

TLDR; ipinfo.io is already used in gluetun to log it out. Since your IP is unlikely to change once you connect, I  can just store the information in memory and serve it through the http api 😉 I'm just low on time, but I'll get to it!

**Comment by ChumpyTheBear** [comment]
Great info. Thanks to both of you!

**Comment by qdm12** [comment]
Done in f9cb71027c17760122a45f82fe88660f206d5d92

Should be built in a few minutes, let me know if it works (haven't tried it, shame on me 😄)

**Comment by nearcatch** [comment]
Thanks, works fine for me. Added to my bash profile.

```shell
# curl via docker container
dcurl() { docker run --name curl_$(date "+%Y-%m-%d_%H-%M-%S.%N") --rm -it --env "TZ=America/Chicago" --network=t2_proxy curlimages/curl "$@"; }
# get ip info in gluetun container(s)
vip() {
  for c in "$@"; do
    echo "${LOG_DARKGREY}$(date "+%Y-%m-%d_%H-%M-%S.%N")${LOG_NORMAL} ${LOG_PURPLE}$c${LOG_NORMAL} $(dcurl -m 15 -f $c:8000/v1/publicip/ip | jq -r '"[0;32m" + .public_ip + "[0m - [1;34m" + .city + ", " + .region + ", " + .country + "[0m"' 2>&1)" | sed 's/, ,/,/'
  done
}
```

Gives this as output when invoked with `vip gluetun`:
```shell
2021-09-06_01-07-45.501679143 gluetun - 104.254.93.195 - Toronto, Ontario, CA
```

**Comment by ChumpyTheBear** [comment]
Works well. Thanks
